mubench.pipeline/tasks/implementations/publish_findings.py

```diff
@@ -72,18 +74,24 @@ def run(self, project: Project, version: ProjectVersion, detector_run: DetectorR
             else:
                 logger.error("%s", e)
 
-    def __slice_by_max_files_per_post(self, potential_hits: List['SpecializedFinding']) -> List[List[Finding]]:
+    def __slice_by_max_files_and_size_per_post(self, potential_hits: List['SpecializedFinding']) -> List[List[Finding]]:
         potential_hits_slice = []
         number_of_files_in_slice = 0
+        size_of_slice = 0
         for potential_hit in potential_hits:
             number_of_files_in_hit = len(potential_hit.files)
-            if number_of_files_in_slice + number_of_files_in_hit > self.max_files_per_post:
+            size_of_hit = self.__get_potential_hit_size(potential_hit)
+            file_count_limit = number_of_files_in_slice + number_of_files_in_hit > self.max_files_per_post
+            file_size_limit = size_of_slice + size_of_hit > self.max_file_size_per_post
+            if potential_hits_slice and (file_count_limit or file_size_limit):
```

Comment:
Was there a particular reason you changed this condition to ensure there's always at least one potential hit in a slice? I like this in general, just curious... Do we have a test that checks we include at least one potential hit, even if both the max size and the max number of files are exceeded?

Reply:
I guess it's not necessary to do this. After the `yield`, we add the potential hit to the next slice anyway. When I added this, I thought we needed this to ensure that we don't silently drop any potential hits. 
Still, I think it makes sense to ensure that we don't return an empty slice. I'll write a test for it.